modules/nodes-pods-vertical-autoscaler-custom.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-vertical-autoscaler-custom_{context}"]
+= Using an alternative recommender 
+
+You can use your own recommender to autoscale based on your own algorithms. If you do not specify an alternative recommender, {product-title} uses the default recommender, which suggests CPU and memory requests based on historical usage. Because there is no universal recommendation policy that applies to all types of workloads, you might want to create and deploy different recommenders for specific workloads. 
+
+For example, the default recommender might not accurately predict future resource usage when containers exhibit certain resource behaviors, such as cyclical patterns that alternate between usage spikes and idling as used by monitoring applications, or recurring and repeating patterns used with deep learning applications. Using the default recommender with these usage behaviors can result in significant over-provisioning and Out of Memory (OOM) kills for your applications. 
```

Comment:
I think that you are supposed to use "might" instead of "can" here to indicate possibility: "...with these usage behaviors can result..."